### PR TITLE
Update ptasieMleczko.md

### PR DESCRIPTION
**Pull request by toobiz**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2679)

### CODE REVIEW

File: Polish/ptasieMleczko.md [hunk 1]
```diff
@@ -10,3 +10,5 @@ Co robisz?
 
 [Chcesz się przebrać](ubranie/ubranie.md)
 
+[Próbujesz uciec](ucieczka/ucieczka.md)
```
Comment (casey-collab) [inline Polish/ptasieMleczko.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**